app/decorators/chart_decorator.rb

```diff
@@ -0,0 +1,22 @@
+class ChartDecorator < Draper::CollectionDecorator
+  def total_by_type_per_day(options)
+    case options[:type]
+    when "Comment"
```

Comment:
This when statement is not being used, but will probably be used in the next implementation. I wrote it more as an example.